code/game/objects/structures/signs.dm

```diff
@@ -12,6 +17,78 @@
 	qdel(src)
 	return
 
+/obj/structure/sign/attackby(obj/item/O, mob/user, params)
+	if(istype(O, /obj/item/weapon/wrench))
+		user.visible_message("<span class='notice'>[user] starts removing [src]...</span>", \
+							 "<span class='notice'>You start unfastening [src].</span>")
+		playsound(src, 'sound/items/Ratchet.ogg', 50, 1)
+		if(!do_after(user, 30, target = src))
+			return
+		playsound(src, 'sound/items/Deconstruct.ogg', 50, 1)
+		user.visible_message("<span class='notice'>[user] unfastens [src].</span>", \
+							 "<span class='notice'>You unfasten [src].</span>")
+		new /obj/item/sign_backing(get_turf(user))
+		qdel(src)
+	if(istype(O, /obj/item/weapon/pen))
+		var/list/sign_types = list("Secure Area", "Biohazard", "High Voltage", "Radiation", "Hard Vacuum Ahead", "Disposal: Leads To Space", "Danger: Fire", "No Smoking", "Medbay", "Science", "Chemistry", \
+		"Hydroponics", "Xenobiology")
+		var/sign_type
+		switch(input(user, "Select a sign type.", "Sign Customization") as null|anything in sign_types)

```

Comment:
I did this for multiple reasons, the first and foremost being that there are a ton of subtypes I would have to remove manually (barsigns...) so unless it's _absolutely_ necessary I'll probably keep it as is.